src/renderer/chrome-api.js

```diff
@@ -99,7 +134,21 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
     chrome.tabs.onRemoved.emit(tabId)
   });
 
+  ipcRenderer.on('cx-event', (_, e) => {
```

Comment:
Agree this legacy method of doing this kind of communcation?

Reply:
refactorized in its own TS module and documented.